pkg/controllers/qat/controller.go

```diff
@@ -77,88 +75,11 @@ func (c *controller) GetTotalObjectCount(ctx context.Context, clnt client.Client
 
 func (c *controller) NewDaemonSet(rawObj client.Object) *apps.DaemonSet {
 	devicePlugin := rawObj.(*devicepluginv1.QatDevicePlugin)
-	yes := true
-	pluginAnnotations := devicePlugin.ObjectMeta.DeepCopy().Annotations
-	return &apps.DaemonSet{
-		ObjectMeta: metav1.ObjectMeta{
-			Namespace:    c.ns,
-			GenerateName: devicePlugin.Name + "-",
-			Labels: map[string]string{
-				"app": appLabel,
-			},
-			Annotations: pluginAnnotations,
-		},
-		Spec: apps.DaemonSetSpec{
-			Selector: &metav1.LabelSelector{
-				MatchLabels: map[string]string{
-					"app": appLabel,
-				},
-			},
-			Template: v1.PodTemplateSpec{
-				ObjectMeta: metav1.ObjectMeta{
-					Labels: map[string]string{
-						"app": appLabel,
-					},
-					Annotations: pluginAnnotations,
-				},
-				Spec: v1.PodSpec{
-					Containers: []v1.Container{
-						{
-							Name:            appLabel,
-							Args:            getPodArgs(devicePlugin),
-							Image:           devicePlugin.Spec.Image,
```

Comment:
watch out for these getting lost

Reply:
Yep, I'm working on new test cases to ensure the result daemonset is the same as it was before using go:embed.